client/src/partials/price_list/pricelist.html

```diff
@@ -1,195 +1,202 @@
+        <div ng-switch-when="create_success">
+          <div class="alert alert-success" id="create_success">
+            <h4>{{ 'PRICE_LIST.SAVE_SUCCESS' | translate }} </h4>
+            </p>
           </div>
+        </div>
 
-          <div class="panel-body">
-            <form name="editPriceList" class="form-horizontal">
-                <div class="form-group">
-                  <label class="col-xs-2 control-label required">{{ "COLUMNS.TITLE" | translate }}</label>
-                  <div class="col-xs-5">
-                    <input class="form-bhima" ng-model="edit.title"  required>
-                  </div>
-                </div>           
-
-                <div class="form-group">
-                  <label class="col-xs-2 control-label">
-                    {{ "COLUMNS.DESCRIPTION" | translate }}
-                  </label>
-                  <div class="col-xs-5">
-                    <input class="form-bhima" ng-model="edit.description">
-                  </div>
-                </div>
-    
-                <input type="submit" ng-disabled="editPriceList.$invalid" class="btn btn-sm btn-success" ng-click="saveMeta()" value="{{ 'FORM.SAVE_CHANGES' | translate }}">
-                <input type="reset" class="btn btn-sm btn-default" ng-click="resetMeta()" value="{{ 'FORM.RESET_FORM' | translate }}">
+        <div ng-switch-when="update_success">
+          <div class="alert alert-success" id="update_success">
+            <h4>{{ 'PRICE_LIST.UPDATE_SUCCESS' | translate }} </h4>
+            </p>
+          </div>
+        </div>
 
-            </form>
+        <div ng-switch-when="delete_success">
+          <div class="alert alert-success" id="delete_success">
+            <h4>{{ 'PRICE_LIST.DELETE_SUCCESS' | translate }} </h4>
+            </p>
           </div>
         </div>
-      </div>
 
-      <div ng-switch-when="add">
-        <div class="panel panel-success">
-          <div class="panel-heading">
-            <b>{{ "PRICE_LIST.ADD" | translate }}</b>
+        <div ng-switch-when="delete_error">
+          <div class="alert alert-danger" id="delete_error">
+            <h4>{{ PriceListCtrl.HTTPError.data.code | translate }} </h4>
+            </p>
           </div>
+        </div>
 
+        <div class="panel panel-primary" ng-switch-when="create">
+          <div class="panel-heading">{{ "PRICE_LIST.ADD" | translate }}</div>
           <div class="panel-body">
-            <form name="newPriceList" class="form-horizontal">
+            <form class="panel-body" name="CreateForm" ng-submit="PriceListCtrl.submit(CreateForm.$invalid)" novalidate>
+
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.label.$invalid }">
+                <label class="control-label">{{ 'COLUMNS.DESIGNATION' | translate }}</label>
+                <input type="text" class="form-control" name="label" ng-model="PriceListCtrl.priceList.label" required>
```

Comment:
Could you put some form validation messages here?  If you use `ng-messages`, they are very easy.